fix english translation (#1739)
996icu/996.ICU: en_US.md

## Gaining more popularity

> In early 2019, a Chinese E-commerce company called _Youzan_ has announced the adoption of the "996" work schedule in the future, at the company's Chinese New Year Party. The CEO of Youzan responds: "This would definitely be a good decision when we look back in a few years time.".

Reviewer: ```suggestion
In early 2019, a Chinese E-commerce company called _Youzan_ announced the adoption of the "996" work schedule in the future, at the company's Chinese New Year Party. The CEO of Youzan responded: "This would definitely be a good decision when we look back in a few years time.".
```

In Mid-March 2019, it was reported that Jingdong started adopting "996" or "995" work schedule in some of the business units. Jingdong PR posted on their maimai(`脉脉`, a Chinese real-name business social network platform) account: "(Our culture is) to devote ourselves with all our hearts (to achieve the business objectives)".

## Compensation and benefits

> According to the Labor Law, employees who follow the "996" work schedule deserve to be paid 2.275 times of their base salary. Unfortunately, people who achieved "996" rarely get paid that much.

Reviewer: ```suggestion
According to the Labor Law, employees who follow the "996" work schedule deserve to be paid 2.275 times of their base salary. Unfortunately, people who work under "996" rarely get paid that much.
```

## Where does the name of the repo `996.ICU` come from?

> If you consistently follow the "996" work schedule, you run the risk of getting yourself into **I**ntensive **C**are **U**nit. 

Reviewer: ```suggestion
If you consistently work under "996" work schedule, you run the risk of getting yourself into **I**ntensive **C**are **U**nit. 
```

Author: solid
